NuGet.Config

```diff
@@ -5,7 +5,6 @@
     <clear />
     <add key="TransportFeed" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
     <add key="templating" value="https://dotnet.myget.org/F/templating/api/v3/index.json" />
-    <add key="dotnet-core" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />
```

Comment:
I'd hold off on this. We are close to make CLR builds successful but until that point I'd keep this one here